jdi-light/src/main/java/com/epam/jdi/light/elements/complex/table/TableMatcher.java

```diff
@@ -5,43 +5,52 @@
 import com.epam.jdi.tools.func.JFunc2;
 import org.openqa.selenium.support.ui.Quotes;
 
-import static com.epam.jdi.light.elements.init.UIFactory.*;
-import static com.epam.jdi.tools.LinqUtils.*;
-import static com.epam.jdi.tools.PrintUtils.*;
-import static java.lang.String.*;
+import static com.epam.jdi.light.elements.init.UIFactory.$$;
+import static com.epam.jdi.tools.LinqUtils.map;
+import static com.epam.jdi.tools.PrintUtils.print;
+import static java.lang.String.format;
 
 /**
  * Created by Roman Iovlev on 26.09.2019
  * Email: roman.iovlev.jdi@gmail.com; Skype: roman.iovlev
  */
 public class TableMatcher {
+    private final String name;
+
     public static JFunc2<BaseTable, TableMatcher[], WebList> TABLE_MATCHER = (table, matchers) -> {
         String locator = format("./%s/ancestor::*/td", print(map(matchers, m ->
-                m.getLocator(table)),"/ancestor::*"));
+                m.getLocator(table)), "/ancestor::*"));
         return $$(locator, table);
     };
+
     public static TableMatcher hasValue(String value, Column column) {
-        return new TableMatcher("/td[%s][normalize-space(.) = "+ Quotes.escape(value)+"]",
-                column, "has '"+value +"'");
+        return new TableMatcher("/td[%s][normalize-space(.) = " + Quotes.escape(value) + "]",
+                column, "has '" + value + "'");
     }
+
     public static TableMatcher containsValue(String value, Column column) {
-        return new TableMatcher("/td[%s][contains(normalize-space(.),"+ Quotes.escape(value)+")]",
-                column, "contains '"+value +"'");
+        return new TableMatcher("/td[%s][contains(normalize-space(.)," + Quotes.escape(value) + ")]",
+                column, "contains '" + value + "'");
     }
 
-    private String locator;
+    private final String locator;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5482899)